cypress/support/e2e.js

```diff
@@ -0,0 +1,4 @@
+import 'cypress-plugin-api'
+import 'cypress-axe'
+import './accessibility_commands'
```

Comment:
Deixe um espaço em branco entre módulos externos e internos.

Isso facilita identificar o que é o que.

```suggestion
import './accessibility_commands'

```